Microsoft.Azure.Cosmos/src/Spatial/GeometryCollection.cs

```diff
@@ -9,12 +9,14 @@ namespace Microsoft.Azure.Cosmos.Spatial
     using System.Collections.ObjectModel;
     using System.Linq;
     using System.Runtime.Serialization;
+    using Microsoft.Azure.Cosmos.Spatial.Converters.STJConverters;
     using Newtonsoft.Json;
 
     /// <summary>
     /// Represents a geometry consisting of other geometries.
     /// </summary>
     [DataContract]
+    [System.Text.Json.Serialization.JsonConverter(typeof(GeometryCollectionSTJConverter))]
```

Comment:
@Pilchie looks like dibahlfi is not available.
I can create a new PR based on this one (mentioning dibahlfi made major work)
I would just go with another naming approach:
1. The converters namespace is `Microsoft.Azure.Cosmos.Spatial.SystemTextJsonConverters`
2. Converters have the same names as NewtonSoft converters

like this
```suggestion
    using Microsoft.Azure.Cosmos.Spatial;
    using Newtonsoft.Json;

    /// <summary>
    /// Represents a geometry consisting of other geometries.
    /// </summary>
    [DataContract]
    [System.Text.Json.Serialization.JsonConverter(typeof(SystemTextJsonConverters.GeometryCollectionConverter))]
```